### PR TITLE
 feat(event): add open call

### PR DESCRIPTION
**Pull request by WikiRik**
Also adds emails to organizers and board members

### CODE REVIEW

File: models/Event.js [hunk 1]
```diff
@@ -501,4 +516,10 @@ Event.beforeValidate(async (event) => {
     if (typeof event.url === 'string') event.url = event.url.toLowerCase().trim();
 });
 
+Event.afterSave(async (event) => {
+    if (event.available_spots <= 0 && event.open_call === true) {
```
Comment (serge1peshcoff) [inline models/Event.js:520]: it can be negative?

Comment (WikiRik) [reply models/Event.js:520]: It shouldn't, so maybe I should change it to === 0

Comment (WikiRik) [reply models/Event.js]: Fixed

**Pull request merged**